.gitignore

```diff
@@ -18,6 +18,3 @@ dist/
 
 # Other
 *.DS_Store
-
```

Comment:
You shouldn't change the .gitignore file in this pr. If you want this changes you should open a new pr.